### ISSUE TITLE
[5.2.*] After lots of research, still having a fatal call. New to Laravel and this is on a *fresh* install

### ISSUE THREAD

**Issue by jbardeleben** [laravel/framework#13080]
Hey all, I have went through every link related to Laravel issues on the web, opened and closed, and I am not finding anyone with this issue. So after a (done this at least 7 times) clean install, every time I try to preview what I'm doing on my laptop, I get this error 

"FatalErrorException in Encrypter.php line 71:
Call to undefined function openssl_encrypt()". 

I have hounded the internet and apparently this is an issue that no one else has had? (I find that hard to believe. Oh, and my "boss" won't give me money for SSL... that alone is not right...)  I am very limited to what I can do (people want to say I should do this or do that. Good luck telling my **** boss that...)

If anyone else has had this issue, please let me know how you fixed it. I'm extremely limited as to what I can do on our "live" server, but my laptop is my dev-soul. If I can't do it here, I would more than guess I'm right that Godaddy or Hostmonster, et-al won't let me. Need some guidance... 

Thanks in advance for your responses.

**Comment by ahmedash95** [comment]
what is your OS and PHP version ?

**Comment by jbardeleben** [comment]
@ahmedash95 Hey, thanks for the quick response! PHP 5.6.15, MySQL 5.7.9, Apache 2.4.9 on Windows 10 (home; I have pro on my work computer, but I haven't tried it on that machine yet as this isn't "work related"). I have been working with PHP for a long time and have never been a big user of frameworks (except Slim for routing), but due to some short deadlines on a couple of side projects, I decided to finally give a framework a run and I'm really liking Laravel after researching what framework I wanted to get into, but this one stumped me. Usually between SO and Google (even Quora to a small extent) I can figure out what ever issue I run into, but this one was just winning me over yesterday, and after reading through a lot of the issues here, I figured there is no way I'm the only one that has run into this. I'm going to dig at it more later on today, if I get it working (my guess is the answer will be in either my Apache or PHP configuration files somewhere), I'll post back with the solution. In the meantime, if you have any guesses or suggestions, I'm all ears! Thanks again for getting back to me

**Comment by GrahamCampbell** [comment]
> Oh, and my "boss" won't give me money for SSL

Unrelated. You just forgot to install the openssl extension, or don't have it enabled for php-fpm.

**Comment by jbardeleben** [comment]
@GrahamCampbell  
Hey Graham,

Now that I have had a few moments to look at my log files, it wasn't that I didn't have openssl loaded, it was that apache wasn't loading the php_openssl.dll, so after making a db backup, I reinstalled wamp on a whim, set up a virtual host, and now it's working with no issues. While at the office today, I was looking at some of the nightmares some have gone through trying to get everything to play nicely with each other and planned on this taking some time, but it was a surprising quick fix... haha. When I did the reinstall of wamp (3.o - apache 2.4.17, php 5.6.15, MySQL 5.7.9 32bit - on windows 10 home) and OpenSSL 1.0.2g, I used the defaults - in case anyone else runs into this issue.

Cheers,
~J
